src/index.html

```diff
@@ -15,8 +16,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
```

Comment:
The `alt` attribute for the logo image is currently 'MOYO logo'. Consider making it more descriptive to improve accessibility, such as 'Company Logo for MOYO'.